app/controllers/catalog_controller.rb

```diff
@@ -58,7 +58,7 @@ def self.modified_field
     config.add_facet_field solr_name("medium", :facetable), label: "Medium", limit: 5
     config.add_facet_field solr_name("language", :facetable), label: "Language", limit: 5
     config.add_facet_field solr_name("division", :facetable), label: "Division", limit: 5
-    config.add_facet_field solr_name("rights", :facetable), label: "Rights", limit: 5
+    config.add_facet_field solr_name("rights", :facetable), helper_method: :license_label, label: "Rights", limit: 5
```

Comment:
Ha, great instinct to worry about retrieval speed from Fedora. In this case it's grabbing the value straight from a [config file](https://github.com/chemheritage/chf-sufia/blob/master/config/authorities/licenses.yml) (via the [questioning_authority](https://github.com/projecthydra-labs/questioning_authority) gem). So I don't think we need caching. That's interesting to see, though, since I've never done that sort of thing before.